docs/how-to/use-layers.md

```diff
@@ -0,0 +1,147 @@
+# How to use layers
+
+Managing multiple services across different environments becomes complex as systems scale. Pebble simplifies this with layered configurations, improving readability and maintainability.
+
+A base layer defines common settings (such as logging), while additional layers handle specific services or environment-specific overrides. This declarative approach, along with delegated layer management, allows for better cross-team collaboration and provides a clear view of each environment's configuration. For example, an operations team could manage base layers for logging, and service teams could manage layers for their services.
```

Comment:
Yeah, I think this is great -- particularly the last sentence really describes it well.